include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -142,6 +239,28 @@ class SplineEvaluator
         return eval(coord_eval, spline_coef);
     }
 
+    /**
+     * @brief Evaluate spline function (described by its spline coefficients) on a mesh.
+     *
+     * The spline coefficients represent a spline function defined on a cartesian product of batch_domain and B-splines
+     * (basis splines). They can be obtained via various methods, such as using a SplineBuilder.
+     *
+     * This is not a multidimensional evaluation. This is a batched 1D evaluation. This means that for each slice of coordinates
+     * identified by a batch_domain_type::discrete_element_type, the evaluation is performed with the 1D set of
+     * spline coefficients identified by the same batch_domain_type::discrete_element_type.
+     *
+     * Remark: calling SplineBuilder then SplineEvaluator corresponds to a spline interpolation.
+     *
+     * @param[out] spline_eval The values of the spline function at the desired coordinates. For practical reasons those are
+     * stored in a ChunkSpan defined on a batched_evaluation_domain_type. Note that the coordinates of the
+     * points represented by this domain are unused and irrelevant (but the points themselves (DiscreteElement) are used to select
+     * the set of 1D spline coefficients retained to perform the evaluation).
```

Comment:
```suggestion
     * @param[out] spline_eval The values of the spline function at the desired coordinates. For practical reasons those are
     * stored in a ChunkSpan defined on a batched_evaluation_domain_type.
```
I would remove the comment on `spline_eval`, I feel it is redundant with the comment in the description of `coords_eval`.